src/components/safe-apps/SafeAppsSDKLink/index.tsx

```diff
@@ -1,49 +1,45 @@
-import { useState } from 'react'
-import Link from 'next/link'
-import Box from '@mui/material/Box'
-import Collapse from '@mui/material/Collapse'
-import Typography from '@mui/material/Typography'
-import MUILink from '@mui/material/Link'
-import Fab from '@mui/material/Fab'
+import { useEffect, useState } from 'react'
+import { Fab, Typography } from '@mui/material'
 import KeyboardDoubleArrowUpRoundedIcon from '@mui/icons-material/KeyboardDoubleArrowUpRounded'
-
+import classnames from 'classnames'
 import CodeIcon from '@/public/images/apps/code-icon.svg'
 import { SAFE_APPS_SDK_DOCS_URL } from '@/config/constants'
 import css from './styles.module.css'
+import ExternalLink from '@/components/common/ExternalLink'
 
 const SafeAppsSDKLink = () => {
-  const [openModal, setOpenModal] = useState<boolean>(false)
+  const [isMini, setMini] = useState(false)
+
+  // Minimize the widget when the user scrolls down
+  useEffect(() => {
+    const MAX_SCROLL = 130
+
+    const onScroll = () => {
+      const isScrolled = document.documentElement.scrollTop > MAX_SCROLL
+      setMini(isScrolled)
+    }
+
+    document.addEventListener('scroll', onScroll)
+
+    return () => document.removeEventListener('scroll', onScroll)
+  }, [])
 
   return (
-    <Box className={css.position} onMouseEnter={() => setOpenModal(true)} onMouseLeave={() => setOpenModal(false)}>
-      <Collapse in={openModal}>
-        {
-          <Box className={css.container}>
-            <CodeIcon />
-
-            <Typography variant="h6" className={css.title}>
-              How to build on Safe?
-            </Typography>
-
-            <Link href={SAFE_APPS_SDK_DOCS_URL} passHref>
-              <MUILink rel="noreferrer noopener" target="_blank" href={SAFE_APPS_SDK_DOCS_URL} className={css.link}>
-                Learn more about Safe Apps SDK
-              </MUILink>
-            </Link>
-          </Box>
-        }
-      </Collapse>
-      <Fab
-        className={css.openButton}
-        variant="extended"
-        size="small"
-        color="secondary"
-        aria-label="add"
-        onClick={() => setOpenModal(false)}
-      >
+    <div className={classnames(css.container, { [css.mini]: isMini })}>
+      <CodeIcon />
+
+      <Typography variant="h6" className={css.title}>
+        How to build on Safe?
+      </Typography>
+
+      <ExternalLink href={SAFE_APPS_SDK_DOCS_URL} className={css.link} noIcon variant="body2">
+        <span>Learn more about Safe Apps SDK</span>
+      </ExternalLink>
+
+      <Fab className={css.openButton} variant="extended" size="small" color="secondary">
```

Comment:
Should we change this to something thats not a button? Currently its still clickable but nothing happens. Could be an issue for keyboard navigation and screen reader users.

Reply:
Good point, I've actually made the button clickable now.